lib/vector-config/src/schema/visitors/unevaluated.rs

```diff
@@ -1,10 +1,26 @@
-//use indexmap::IndexMap;
+#![allow(warnings)]
```

Comment:
What was warning in this file?

Reply:
Oh, woops. Looks like I left a bunch of stuff around that I added to reduce friction during development. I'll clean this up.

Reply:
Resolved in 310846b7b, including a bunch of cleanup of dead code, etc, that it was hiding.